src/app/static/src/app/apiService.ts

```diff
@@ -132,10 +137,12 @@ export class APIService<S extends string, E extends string> implements API<S, E>
                 data: {}
             }
         }
+         */
 
-        if (!isHINTResponse(failure)) {
+        if(!isHINTResponse(failure)){
             this._commitError(APIService.createError("apiCouldNotParseError"));
```

Comment:
Does your average user understand what this error means? Would plain English like: "You are not currently logged in", not be clearer

Reply:
This is just a whitespace change, not related to this ticket.